### ISSUE TITLE
x/tools/go/gccgoexportdata: escape tag is not supported

### ISSUE THREAD

**Issue by cherrymui** [golang/go#23324]
With gccgo, when the escape analysis is turned on (`-fgo-optimize-allocs`), it writes escape tag for function arguments in the export data, with the format `<esc:0xNN>` before the `type` tag, like
```
func Compare (a <esc:0x1> <type 19 [] <type -20>>, b <esc:0x1> <type 19>) <type -11>;
```

The `gccgoexportdata` package cannot parse export data with escape tags in them.

```
reading export data: /usr/local/google/home/cherryyz/w/gccgo2/lib64/go/8.0.0/x86_64-pc-linux-gnu/bytes.gox: import error bytes:11:15 (byte offset = 516): expected keyword type, got "esc"
```

cc @ianlancetaylor @alandonovan

**Comment by griesemer** [comment]
@cherrymui Is this a blocker for you? Let me know and I'll prioritize if necessary.

**Comment by cherrymui** [comment]
@griesemer, thank you! It is a blocker for turning on the escape analysis by default for gccgo. Meanwhile we could use the flag `-fgo-optimize-allocs` to turn on the escape analysis, but some tools would be fail to work.

**Comment by griesemer** [comment]
We have a gccgoimporter also in the std library. This seems a very safe fix that we should probably add to Go 1.10.
